src/components/BookList.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 
-const BookList = () => (
-  <div>
-    <Book />
-    <Book />
-    <Book />
-  </div>
-);
+const BookList = () => {
+  const myState = useSelector((state) => state);
```

Comment:
- [ ] I would recommend mentioning which state you want to get in useSelector. In BookList we need to get the state of the books array: ```useSelector((state) => state.bookReducer)```. For example if we need to get categories we need to write: ```useSelector((state) => state.catReducer)```.